src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/TreeViewTests.cs

```diff
@@ -4722,6 +4722,19 @@ public void TreeViewNodeSorter_Set_GetReturnsExpected(IComparer value)
         Assert.Same(value, treeView.TreeViewNodeSorter);
     }
 
+    [WinFormsFact]
+    public void TreeViewNodeSorter_Set_SortedFalseIfNull()
+    {
+        using var treeView = new TreeView
```

Comment:
Prefer target type new:
```suggestion
        using TreeView treeView = new()
```